textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/source/IndexSearchSourceOperatorTest.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.dataflow.source;
+
+import java.text.ParseException;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Assert;

```

Comment:
Importing necessary assert functions instead of the entire `Assert` class is better I think. For example, if only `assertEquals` is being used only that should be imported.